### DISCUSSION TITLE
View redemptions of discount codes

### DISCUSSION THREAD

**Discussion by KatrineStub** [medusajs/medusa#4396] [Feature Requests]
We would greatly benefit from having a clear summary of orders where a specific discount code has been used when performing a search. Unfortunately, our current system lacks the capability to provide a comprehensive overview of orders associated with a particular discount code. As a result, we face limitations in monitoring and investigating any suspicious activities that may be linked to the use of discount codes.

**Comment by srindom** [comment]
Would adding an "Orders" section here work where you can see all the orders that have used a given discount - or what would be the most helpful approach? 


![CleanShot 2023-06-23 at 3 33 34@2x](https://github.com/medusajs/medusa/assets/7554214/39d932b6-6f5a-4dc0-b7e4-4c64776e5bf7)

**Reply by KatrineStub** [reply]
Yes, that would be a great help

**Comment by srindom** [comment]
Also, what are you looking for when investigating "suspicious activities"?

**Reply by KatrineStub** [reply]
Potential misuse of discount codes. We see that our discount codes are being shared online and are misused. It would be nice to potentially be able to cancel these orders

**Reply by tsvetann** [reply]
@KatrineStub what do you mean shared and misused? Are your discount codes public? If they are what's the misuse here?

**Reply by magdafiktus** [reply]
no, some of our codes are internal and they are meant to be used only by some, hence we need more visibility on the usage :)
